Checkr/src/components/organisms/PreAdverse/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+//import theme from '../../../components/themes/index';
+import Layout from '../../../components/templates/Layout/index';
+import Buttons from '../../../components/atoms/Button/index';
+import { useState } from 'react';
+import PreviewModal from '../PreviewModal';
+
+ const PreAdverse = () => {
+    const [isCheckboxChecked, setIsCheckboxChecked] = useState(false);
+    const [isModalOpen, setIsModalOpen]  = useState(false);
+
+    const handleCheckboxChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setIsCheckboxChecked(event.target.checked);
+      };
+
+      const handlePreviewModalOpen = () => {
+        setIsModalOpen(true);
+      };
+    
+
+    return (
+    <Layout>
+    <div style={{
+        maxWidth: '900px',
+        // margin: '0 auto',
+        border: '1px solid #ccc',
+        padding: '20px',
+        marginTop: '30px',
+        justifyItems: 'left',
+        alignItems: 'left',
+        boxShadow: '0px 0px 10px rgba(0, 0, 0, 0.1)',
+        justifyContent: 'left',
+        color: 'gray',
+    }}>
+    
+        <b>From:</b> Kyle@Checkr.com
+        <hr/>
+        <b>To:</b> John.Smith@Checkr.com
+        <hr/>
+        <b>Subject:</b> Pre-Adverse Action Notice - Checkr-Bpo
+        <hr/>
+        <br/>
+        Dear John Smith,
```

Comment:
Remove this content from the component, store it in a variable outside the component.